src/test/storage.zig

```diff
@@ -55,8 +55,10 @@ pub const Storage = struct {
         /// Tick at which this read is considered "completed" and the callback should be called.
         done_at_tick: u64,
 
-        fn less_than(storage: *Read, other: *Read) math.Order {
-            return math.order(storage.done_at_tick, other.done_at_tick);
+        fn less_than(context: void, a: *Read, b: *Read) math.Order {
+            _ = context;
```

Comment:
I feel like for arguments of type void like this it would be nicer just to not give them a name, no need to explicitly ignore nothing :D
```suggestion
        fn less_than(_: void, a: *Read, b: *Read) math.Order {
```